examples/vpc_service_control/README.md

```diff
@@ -0,0 +1,129 @@
+        - `shared_secret_string_for_vpn_connection`: *same* as in step 1
+        - Leave the rest blank.
+    - From a command line terminal:
+        - `cd ../vpc_sc_project`
+        - `terraform plan`
+            - It should show a plan to create one project, one network, one subnet, one firewall rule, one route.
+        - `terraform apply`
+            - You might not get the "Compute Engine API" error message here, but if you do, enable it and try again.
+
+3. Reserve a public IP for your "on-prem" VPN router (to be clear, further use of "on-prem" in these instructions refers to the GCP project that we're pretending is your on-prem environment).
+    - Open `onprem_project/main.tf` in your favorite text editor.
+    - Search for `STEP 1A`.
+    - Don't actually delete the line; just add `*/` to the **end** of the line.
+    - Search for `STEP 1B`.
+    - Don't actually delete the line; just add `/*` to the **beginning** of the line.
+    - Save file.
+    - `cd onprem_project/`
+    - `terraform plan` and make sure it says it's going to reserve an IP address
+    - `terraform apply`
+    - Write down the public IP address that is printed on the terminal when the command succeeds.
+    - **Important:**  Insert the public IP address into `vpc_sc_project/terraform.tfvars` under `ip_addr_of_onprem_vpn_router`.
+
+4. Reserve a public IP for your VPC Service Control project's VPN router.
+    - Follow all the steps in step 3, but instead do it in `vpc_sc_project/main.tf` and search for `STEP 2A` and `STEP 2B`.
+    - Write down the public IP address that is printed on the terminal when the command succeeds.
+    - **Important:**  Insert the public IP address into `onprem_project/terraform.tfvars` under `ip_addr_of_cloud_vpn_router`.
+    - Now the two VPN routers in each project will know how to find each other.
+    
+5. Spin up the VPN router, Cloud Router, and VPN tunnel in on-prem project.
+    - Edit `onprem_project/main.tf`
+    - Delete `STEP 3A` and `STEP 3B` lines (comment them instead of actually deleting for best results).
+    - `cd onprem_project/`
+    - `terraform plan`
+    - `terraform apply`
+
+6. Spin up the VPN router, Cloud Router, and VPN tunnel in the VPC Service Control project.
+    - Same as step 5, but with `vpc_sc_project/main.tf` and `STEP 4A` and `STEP 4B`
+    - Once this is done, open the Cloud Console for either project.  Go to the Hybrid Connectivity -> VPN section, and click on the VPN tunnel you created.  You should see the connection established to the other VPN router, and it should say that the BGP session is established.
+
+7.  Continue bouncing back and forth between the two `main.tf` files, uncommenting code blocks in numeric order for `STEP 5A`, `6A`, `7A`, and running `terraform apply` after uncommenting each code block.
+    - `7A` is in `vpc_sc_project/main.tf`.  It will prompt you to enable the Cloud DNS API.  Enable it.
+
+TODO: Complete the remaining instructions with more detail.
```

Comment:
It appears that this guide is not complete.

Reply:
It's complete.  This was a suggestion to myself to add more detail, but I think it's sufficient.